recipes/python.rb

```diff
@@ -0,0 +1,2 @@
+node.override['python']['pip_location'] = \
+"#{node['python']['prefix_dir']}/bin/pip2.7"
```

Comment:
Yes. When I do that I get the error:

``The executable python2.7 (from --python=python2.7) does not exist``